src/main/java/ui/screens/BurndownScreen.java

```diff
@@ -75,13 +75,33 @@ protected void afterVisualizationMount() {
             updateDisplayedSprints(overlayCheckBox.isSelected());
         });
 
+        sprint_name.textProperty().bind(Bindings.createStringBinding(() -> {
+            List<Sprint> sprints = sprintMultiselect.getSelectionModel().getSelectedItems();
+            String sprint = "";
+            if (sprints.size() == 0) {
+                sprint = "(No Sprint Selected)";
+            }
+            if (sprints.size() == 1) {
+                sprint = sprints.get(0).getName();
+            }
+            else if (sprints.size() > 1) {
+                sprint = "Showing ";
+                for(Sprint s : sprints) {
+                    sprint += (s.getName() + ", ");
+                }
+            }
```

Comment:
An extra comma gets added at the end. A more robust way to do this might be to use a stream and concatenate everything with `Collectors.joining(", ")`. The resulting code would look something like `sprints.stream().map(Sprint::getName).collect(Collectors.joining(", "));`, eliminating the need for the if-else if statement.

Reply:
Done